pkg/utils/configMapUtils.go

```diff
@@ -1,9 +1,22 @@
 package utils
 
 import (
+	log "github.com/sirupsen/logrus"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func (expDetails *ExperimentDetails) PatchConfigMaps(clients ClientSets) error {
+	log.Infof("Finding configMaps in the chaosExperiments")
+	expDetails.SetConfigMaps(clients)
+	log.Infof("Validating ConfigMaps")
```

Comment:
validating configmaps specified in the chaos experiment

Reply:
Already going it individually